doc/api/child_process.md

```diff
@@ -1434,6 +1434,32 @@ console.log(`Spawned child pid: ${grep.pid}`);
 grep.stdin.end();
 ```
 
+### `subprocess.hasRef()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
```

Comment:
```suggestion

> Stability: 1 - Experimental

```
Should this start off with experimental stability?

Reply:
Applied. Thanks for the heads up.